addons/sale_timesheet/models/project.py

```diff
@@ -200,15 +200,13 @@ def _compute_allocated_hours(self):
         uoms_dict = {uom.id: uom for uom in self.env['uom.uom'].browse(uom_ids)}
 
         for project in self:
-            if not project.sale_line_id:
```

Comment:
Couldn't we rephrase the commit message as such?
```
[FIX] sale_timesheet: fix the computation of allocated_hours on project
Before this commit:
- When project is created from SO, the quantity of SOL is set in allocated hours.
  Then, when navigating to the 'invoicing' tab and adding an employee mapping,
  the `allocated hours` is reset to 0.
- When manually removing the SOL of a project, the allocated_hours is reset to 0.

After this commit, the above issues have been fixed.

task-2788878
```

Reply:
@IT-Ideas , done!!. thanks